### ISSUE TITLE
WaybackProxy 

### ISSUE THREAD

**Issue by maakuth** [zouppen/retroska#23]
For content better suited for 90s browsers compared to #16, adding support for [WaybackProxy](https://github.com/richardg867/WaybackProxy) would be nice.

**Comment by zouppen** [comment]
Good idea. Instead of adding it to the current monolithic container, we should probably run it as its own container.

Some host-side user-space tool (just a script or something more fancy) for selecting the services would be nice, so user could enable this and get it automatically set up with non-conflicting IP addresses.
